charts/mtav6.x/templates/01-sonataflow_mta-analysis-v6.yaml

```diff
@@ -0,0 +1,537 @@
+          useResults: true
+        functionRef:
+          arguments:
+            application:
+              id: .application.id
+              name: .application.name
+            kind: .trackerProjectIssueTypes[0].id
+            parent: .trackerProject.id
+            tracker:
+              id: .tracker.id
+              name: .tracker.name
+          invoke: sync
+          refName: createTicket
+        name: CreateTickets
+      name: CreateJira
+      transition:
+        nextState: NotifyCompletion
+      type: operation
+    - actionMode: sequential
+      actions:
+      - actionDataFilter:
+          useResults: true
+        condition: .backstageUser != null or .backstageGroup != null
+        functionRef:
+          arguments:
+            actions:
+            - title: View report
+              url: $SECRET.mta.url + "/applications/" + (.application.id|tostring)
+            message: '"MTA analysis is completed and the report for " + .application.repository.url
+              + " is ready."'
+            origin: MTA Analysis Workflow
+            targetGroups: if .backstageGroup != null then [.backstageGroup] else []
+              end
+            targetUsers: if .backstageUser != null then [.backstageUser] else [] end
+            title: MTA analysis completed
+            topic: MTA Analysis Workflow
+          invoke: sync
+          refName: createNotification
+      name: NotifyCompletion
+      transition:
+        nextState: report
+      type: operation
+    - actionMode: sequential
+      actions: null
+      end:
+        terminate: true
+      name: report
+      stateDataFilter:
+        output: '{mtaAnalysisResultURL: ($SECRET.mta.url + "/hub/applications/" +
+          (.application.id|tostring) + "/bucket/windup/report/"), migrationWaveJiraTicketURL:
+          .migrationWaveJiraTicket.link, repositoryURL: .repositoryURL, workflowOptions:
+          {migrationOptions: [{id: "m2k", name: "Move2Kube workflow"}]}}'
+      type: operation
+    - actionMode: sequential
+      actions:
+      - actionDataFilter:
+          useResults: true
+        condition: .backstageUser != null or .backstageGroup != null
+        functionRef:
+          arguments:
+            message: '"MTA analysis for " + .application.repository.url + " failed.
+              Check logs of task pod: " + .taskgroup.tasks[0].pod'
+            origin: MTA Analysis Workflow
+            targetGroups: if .backstageGroup != null then [.backstageGroup] else []
+              end
+            targetUsers: if .backstageUser != null then [.backstageUser] else [] end
+            title: MTA analysis failed
+            topic: MTA Analysis Workflow
+          invoke: sync
+          refName: createNotification
+      end:
+        terminate: true
+      name: NotifyFailureBackstage
+      type: operation
+    timeouts:
+      workflowExecTimeout:
+        duration: PT1H
+  podTemplate:
+    container:
+      resources: {}
```

Comment:
the image should be specified here, in a similar way as done for the existing mta workflow:
https://github.com/parodos-dev/serverless-workflows-config/blob/main/charts/mta/templates/01-sonataflow_mtaanalysis.yaml#L252

so it appears the repository was never created and the image wasn't pushed: https://github.com/parodos-dev/serverless-workflows/actions/workflows/mta-v6.x.yml

I'm going to create the repositories in quay for both workflows (v6 and v7) and repeat the failed actions.
Only then this CR can be updated.

Reply:
now things looks better - there is an image created successfully at `quay.io/orchestrator/serverless-workflow-mta-v6.x:1bde42520bd287ded1eab55380a2d17912f0f32a`
and it needs to be added here.

then, by merging this PR, the next failed task in the job below should be resolved:
 https://github.com/parodos-dev/serverless-workflows/actions/runs/9552838333/job/26330499375

Reply:
in the meantime, running the build for mta-v7